test_suite/generic/test_generic.py

```diff
@@ -396,6 +399,7 @@ def test_wheel_group_not_set_to_default_user(self, host):
 
 
 @pytest.mark.pub
+@pytest.mark.order(3)
```

Comment:
Do we know how the pytest-order would work with parallel execution of tests? (Multiple ssh connection wo the same instance)

Reply:
It will always respect the ordering, even though there are multiple threads.